src/main/java/pages/BasePage.java

```diff
@@ -24,6 +24,12 @@ public class BasePage {
     protected final String footersocialXpath = "//section[@class='footersocial']";
     protected final String footerlinksXpath = "//section[@class='footerlinks']";
     public By searchInputLocator = By.id("filter_keyword");
+    public By facebookHeaderButtonLocator = By.xpath("//header//a[contains(@class, 'facebook')]");
+    public By twitterHeaderButtonLocator = By.xpath("//header//a[contains(@class, 'twitter')]");
+    public By linkedinHeaderButtonLocator = By.xpath("//header//a[contains(@class, 'linkedin')]");
+    public By facebookFooterButtonLocator = By.xpath("//footer//a[contains(@class, 'facebook')]");
+    public By twitterFooterButtonLocator = By.xpath("//footer//a[contains(@class, 'twitter')]");
+    public By linkedinFooterButtonLocator = By.xpath("//footer//a[contains(@class, 'linkedin')]");
```

Comment:
@marikacode duplicated locators

Reply:
@jakub-krzysiek 
It's not duplicated. One is for the footer the other is for the header